.github/ISSUE_TEMPLATE/virtual_meetup_proposal.md

```diff
@@ -9,6 +9,19 @@ assignees: clairernovotny, jongalloway, sarajchipps, jamesmontemagno
 ### Title
 Your session name here
 
+###  User Group Association (if any)
+* Name: Seattle Mobile .NET (Xamarin) User Group
+* URL: https://www.meetup.com/SeattleMobileDevelopers/
+
+### Streaming Support
+* [ ] Hosting, streaming, & archiving on .NET Foundation YouTube
+* [ ] Allowed to make public after stream
+
+Fill in [x] or leave [ ] if not needed.
```

Comment:
```suggestion
### Streaming Support
[ ] Live streaming to the .NET Foundation YouTube channel

Fill in [x] or leave [ ] if not needed.

> If not checked, we will:
> 1. Promote your event to the Virtual Meetup members.
> 2. If you post your session video to YouTube, we'll add it to the .NET Foundation Virtual Event playlist.

> If checked, we will:
> 1. Provide you a live streaming link to provide to your meetup members.
> 2. Live stream your event to the .NET Foundation YouTube channel ([more information here](streaming-info.md)). 
> 3. When complete, your event video will be added to the .NET Foundation Virtual Event playlist.
```

Reply:
I think we need to be more clear on what we'll do in either case. Also, we can set the policy that we will publish all Virtual User Group events by default, I don't think we need to ask for separate permission for each event. If someone needs to make an exception that's fine, but by default our policy should be to publish them.